tests/rptest/services/openmessaging_benchmark.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import re
+import threading
+import os
+import json
+
+from ducktape.services.service import Service
+from ducktape.utils.util import wait_until
+
+
+# Benchmark worker that is used by benchmark process to run consumers and producers
+class OpenMessagingBenchmarkWorker(Service):
+    PERSISTENT_ROOT = "/var/lib/openmessaging"
+    STDOUT_STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "worker.log")
+    PORT = 9090
+    STATS_PORT = 9091
+
+    logs = {
+        "open_messaging_benchmark_worker_stdout_stderr": {
+            "path": STDOUT_STDERR_CAPTURE,
+            "collect_default": True
+        }
+    }
+
+    def __init__(self, ctx, num_workers=3):
+        super(OpenMessagingBenchmarkWorker,
+              self).__init__(ctx, num_nodes=num_workers)
+
+    def start_node(self, node):
+        self.logger.info("Starting Open Messaging Benchmark worker node on %s",
+                         node.account.hostname)
+
+        self.clean_node(node)
+
+        node.account.mkdirs(OpenMessagingBenchmarkWorker.PERSISTENT_ROOT)
+
+        start_cmd = f"cd /opt/openmessaging-benchmark; \
+                      bin/benchmark-worker \
+                      --port {OpenMessagingBenchmarkWorker.PORT} \
+                      --stats-port {OpenMessagingBenchmarkWorker.STATS_PORT} \
+                      >> {OpenMessagingBenchmarkWorker.STDOUT_STDERR_CAPTURE} 2>&1 & disown"
+
+        with node.account.monitor_log(
+                OpenMessagingBenchmarkWorker.STDOUT_STDERR_CAPTURE) as monitor:
+            node.account.ssh(start_cmd)
+            monitor.wait_until(
+                "Javalin has started",
+                timeout_sec=100,
+                backoff_sec=10,
+                err_msg=
+                "Open Messaging Benchmark worker service didn't finish startup"
+            )
+        self.logger.info(
+            f"Open Messaging Benchmark worker is successfully started on node {node.account.hostname}"
+        )
+
+    def stop_node(self, node):
+        self.logger.info(
+            f"Stopping Open Messaging Benchmark worker node on {node.account.hostname}"
+        )
+        node.account.kill_process("benchmark", allow_fail=False)
```

Comment:
I hope that we are able to run more than one test in parallel on a cluster in which case having a generic name like 'benchmark' might conflict with other tests. Maybe we can rename it? This could also be a follow up patch...